src/login/logind-dbus.c

```diff
@@ -1017,6 +1011,12 @@ static int create_session(
 
         session->create_message = sd_bus_message_ref(message);
 
+        /* Now call into session_send_create_reply(), which will reply to this call. Or it won't if we
+         * spawned a user service manager and/or a sessions scope, and it's not ready yet. */
+        r = session_send_create_reply(session, /* error= */ NULL);
+        if (r < 0)
+                return r;
+
         /* Now, let's wait until the slice unit and stuff got created. We send the reply back from
          * session_send_create_reply(). */
```

Comment:
This comment should be more specific? I.e. mention that we call this here because for sessions needing no scope and whose user's manager is already active, there are no pending jobs, so we should send the reply immediately (`match_job_removed` is not involved). It took me a while to understand this.

Reply:
uh, that comment should be removed actually, replaced by the the one this patch adds further up.

Reply:
(but i'll reword it a bit too)